frontend/src/components/users/Grid.test.tsx

```diff
@@ -6,22 +6,23 @@ import {
   makeWidgetCoordinates,
 } from './GridTools'
 import { mock } from 'dummy'
+import { pos } from 'vec2'
 
 test.each(
   // prettier-ignore
-  [{ widget: mock[0], res: [{ x: 0, y: 0 }] },
-  { widget: mock[1], res: [{ x: 1, y: 0 }, { x: 2, y: 0 }] },
-  { widget: mock[2], res: [{ x: 0, y: 1 }, { x: 0, y: 2 }] },
-  { widget: mock[3], res: [{ x: 3, y: 1 }, { x: 3, y: 2 }, { x: 4, y: 1 }, { x: 4, y: 2 }] }]
+  [{ widget: mock[0], res: [pos(0, 0)] },
+  { widget: mock[1], res: [pos(1, 0), pos(2, 0)] },
+  { widget: mock[2], res: [pos(0, 1), pos(0, 2)] },
+  { widget: mock[3], res: [pos(3, 1), pos(3, 2), pos(4, 1),pos(4, 2)] }]
 )('makeWidgetCoordinates($widget) -> [$res]', ({ widget, res }) =>
   expect(makeWidgetCoordinates(widget)).toEqual(res)
 )
 
 test.each(
   // prettier-ignore
-  [{ start: { x: 0, y: 0 }, end: { x: 1, y: 1 }, res: [mock[0]] },
-  { start: { x: 0, y: 0 }, end: { x: 2, y: 2 }, res: [mock[0], mock[1], mock[2], mock[4]] },
-  { start: { x: 2, y: 2 }, end: { x: 3, y: 3 }, res: []}]
+  [{ start: pos(0, 0 ), end: pos(1, 1), res: [mock[0]] },
+  { start: pos(0, 0), end: pos(2, 2), res: [mock[0], mock[1], mock[2], mock[4]] },
+  { start: pos(2, 2), end: pos(3,3), res: []}]
```

Comment:
```suggestion
  { start: pos(2, 2), end: pos(3, 3), res: []}]
```
여기도 prettier 적용 누락인 것 같습니다

Reply:
위에 // prettier-ignore 주석이 있고 해제하면 24번 라인이 안예쁘게 나오네요
주석은 안 풀고 공백만 추가하겠습니다